content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -7,19 +7,31 @@ This document describes how to install SODA projects in a local cluster with det
 
 `Hereafter, Hotpot refers to API, Controller and Dock projects`
 
-### Pre-config (Ubuntu 16.04 or Ubuntu 18.04)
-All the installation work is tested on `Ubuntu 16.04` and `Ubuntu 18.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+{{% notice info %}}
+**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
+**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
```

Comment:
when its deprecated , why support deprecated OS . Even from ubuntu, its going to be EOL.

Reply:
Ubuntu 16.04 LTS is widely used and there will be cases where the OS will not be upgraded.
in this case we will let them know of the issues that could be faced and help with installation.